src/main/resources/Diagram/DiagramViewSheet.xml

```diff
@@ -40,7 +40,7 @@
   <syntaxId>xwiki/2.1</syntaxId>
   <hidden>true</hidden>
   <content>{{velocity}}
-#if ($doc.getObject('Diagram.DiagramClass'))
+#if ($doc.getObject('Diagram.DiagramClass') || $request.export != '')
```

Comment:
Better use the *silent* notation here (because ``null`` is different than the empty string):
```suggestion
#if ($doc.getObject('Diagram.DiagramClass') || "$!request.export" != '')
```